serenity-reports/src/main/java/net/thucydides/core/reports/html/Reporter.java

```diff
@@ -39,11 +39,13 @@ private void generateReports() {
             final List<ReportExecutor> partitions
                     = reportingTasks.stream()
                     .map(ReportExecutor::new)
+                    .filter(data-> data != null)
                     .collect(Collectors.toList());
 
             final List<ReportExecutorFuture> futures
                     = partitions.stream()
                     .map( partition -> new ReportExecutorFuture(executorPool.submit(partition), partition.getReportingTask()) )
+                    .filter(data-> data != null)
```

Comment:
This appears to check that the result of the `new()` is not null, which, in theory at least, should never be possible in Java.

Reply:
Issue:- Failed to generate report #1568  This is the test case which causes this bug having null values in list
when i tried to debug it this showed null variables in list